apps/api/src/lesson/lesson.type.ts

```diff
@@ -1,5 +1,5 @@
 export const LESSON_TYPES = {
-  TEXT: "text",
+  TEXT: "text_block",
```

Comment:
We don't need it, we can use `text`

Reply:
Ok, i will change it back, and adjust on frontend to be also "text"